package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -317,32 +308,37 @@ class HydrogenBondAnalysis_OtherFF(HydrogenBondAnalysis):
           11. "distance"
           12. "angle"
 
-      It takes up more space than
-      :attr:`~HydrogenBondAnalysis.timeseries` but it is easier to
-      analyze and to import into databases (e.g. using recsql_).
+      It takes up more space than :attr:`~HydrogenBondAnalysis.timeseries` but
+      it is easier to analyze and to import into databases or dataframes.
+
+
+      .. rubric:: Example
+
+      For example, to create a `pandas.DataFrame`_ from ``h.table``::
+
+         import pandas as pd
+         df = pd.DataFrame.from_records(h.table)
 
       .. Note::
 
-         The *index* is a 1-based index. To get the :attr:`Atom.index` (the
-         0-based index typically used in MDAnalysis simply subtract 1. For
-         instance, to find an atom in :attr:`Universe.atoms` by *index* one
-         would use ``u.atoms[idx_zero]``. The 1-based index is deprecated and
-         targeted for removal in 0.16.0
+         Each index variable named *idx* is a 1-based index. To get the :attr:`Atom.index` (the
+         0-based index typically used in MDAnalysis) simply subtract 1, or better, use the 0-based variables named *index*.
 
+         For instance, to find an acceptor atom in :attr:`Universe.atoms` by *index* one
+         would use ``u.atoms[acceptor_index]``.
 
+      .. deprecated:: 0.15.0
```

Comment:
Can you somewhere add a developer note how to remove these things. I must have missed it looking for deprecations to remove before the last release